js/lib/Utils.js

```diff
@@ -384,6 +384,8 @@ Utils.loadData = function(params) {
         throw "Utils.loadData called without a callback. It must have a callback to work.";
     }
 
+    if (ilib._getPlatform() === "qt") ilib._cacheMerged = true;
+
```

Comment:
All direct manipulation of the ilib name space should be contained within the ilib.js file as much as possible.